README.md

```diff
@@ -20,3 +21,24 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+## Answer
+
+Finding the runtime complexity:
+At the beginning of each call of permutationSort there is a call of isSorted to check the permutation. The isSorted function has a complexity of $\Theta(n)$.
+During the first call of permutationSort, the primary for loop runs $n$ times. Within the for loop there is a swapping function that has constant complexity, a recursive call to permutationSort that will handle 1 less element, and a call of the isSorted function. Combining this we get the recurrance relation:
+
+$$\begin{equation}T(n)=\begin{cases}1, & \text{if $n<=1$} \\
+n+n^2*T(n-1)+C, & \text{if $n>1$}.\end{cases}\end{equation}$$
+
+Ignoring the low order terms, we can observe that this relation works out to be $n^2(n-1)^2(n-2)^2...T(n-i)$.
+Thus, the highest order term will be $n^2!$; the runtime complexity of this algorithm is $\Theta(n^2!)$.
```

Comment:
Hmm, so you're doing a quadratic amount of work for each permutation? That doesn't sound right, at least it seems like you'd be doing more work than you need to.